testsuite/tests/conftest.py

```diff
@@ -333,7 +333,7 @@ def threescale(testconfig):
         testconfig["threescale"]["admin"]["url"],
         testconfig["threescale"]["admin"]["token"],
         ssl_verify=testconfig["ssl_verify"],
-        wait=16
+        wait=0
```

Comment:
Do we really need this parameter? Cannot we just omit it?

Reply:
No, no param means no waiting, `wait=0` means "wait until condition is met" `wait=16` means "wait next 16 seconds after condition is met". `wait=0` behavior is desirable and no waiting is reasonable as library default, therefore this custom value here